stream_alert/athena_partition_refresh/main.py

```diff
@@ -497,16 +500,19 @@ def _delete_messages_from_queue():
                          for message in message_batch])
 
             # Handle successful deletions
-            self.deleted_messages += len(resp['Successful'])
-
+            if resp.get('Successful'):
+                self.deleted_messages += len(resp['Successful'])
             # Handle failure deletion
-            if resp.get('Failed'):
+            elif resp.get('Failed'):
```

Comment:
Good call - `Successful` may not be defined. However, both `Successful` and `Failed` can be defined, this should just be a separate `if` statement